go.mod

```diff
@@ -4,6 +4,12 @@ go 1.12
 
 require (
 	github.com/aws/aws-sdk-go v1.26.6
-	github.com/mattermost/mattermost-server/v5 v5.18.0
-	github.com/pkg/errors v0.8.1
+	github.com/blang/semver v3.5.1+incompatible
+	github.com/mattermost/ldap v3.0.4+incompatible // indirect
+	github.com/mattermost/mattermost-server/v5 v5.21.0
+	github.com/onsi/ginkgo v1.8.0 // indirect
+	github.com/onsi/gomega v1.5.0 // indirect
+	github.com/pkg/errors v0.9.1
+	github.com/stretchr/testify v1.5.1 // indirect
+	gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d // indirect
```

Comment:
I'm wondering where these indirect dependencies come from. Maybe just remove the manually?

Reply:
Thanks, now updated.